inline-c-cpp/inline-c-cpp.cabal

```diff
@@ -57,11 +89,6 @@ test-suite tests
                      , template-haskell
                      , vector
   default-language:    Haskell2010
-  ghc-options:
-    -optc-std=c++11
-  if os(darwin)
-    ghc-options: -pgmc=clang++
-  extra-libraries:     stdc++
-  cc-options:          -Wall -Werror -optc-xc++ -std=c++11
```

Comment:
Restoring `-Werror` is feasible but may require duplicating a large part of `common cxx-opts`